packages/backend/src/config/config.production.ts

```diff
@@ -58,6 +58,7 @@ export function getProductionConfig(): Config {
     tvl: {
       alchemyApiKey: getEnv('ALCHEMY_API_KEY'),
       etherscanApiKey: getEnv('ETHERSCAN_API_KEY'),
+      arbiscanApiKey: getEnv('ARBISCAN_API_KEY'),
```

Comment:
remember you will have to set it on Heroku in advance, otherewise the main Application process will fail